app/policies/linked_profile_policy.rb

```diff
@@ -0,0 +1,11 @@
+class LinkedProfilePolicy < ApplicationPolicy
+  def show?
+    record.user == user || record.public? || is_admin?
+  end
```

Comment:
With Pundit-Resources we no longer use `#show?`

Instead, add a `Scope` class like you see in the `LibraryEntryPolicy`.  The logic should be pretty similar.